.travis.yml

```diff
@@ -55,13 +55,13 @@ before_install:
 
 # setup requirements for running unit/behat tests
 install:
-  - if [ "$CHECK_CS" != "" ] ; then travis_retry composer install --prefer-dist --no-interaction --no-suggest ; fi
-  - if [ "$TEST_CONFIG" != "" ] ; then ./bin/.travis/prepare_unittest.sh ; fi
-  - if [ "$BEHAT_OPTS" != "" ]; then ./bin/.travis/prepare_ezplatform.sh ; fi
+  - travis_retry composer install --prefer-dist --no-interaction
+  - if [ "${BEHAT_OPTS}" != "" ]; then ./vendor/bin/prepare_project_edition.sh oss ${PROJECT_VERSION} ${COMPOSE_FILE} ; fi
+  - if [ "$BEHAT_OPTS" != "" ] ; then cd "$HOME/build/project"; docker-compose exec --user www-data app sh -c "mkdir -p config/packages/behat/" ; fi
```

Comment:
Right now this directory does not exist - I create it manually for these tests to pass, we might add a recipe that creates it in the future (but it's a bit too early IMHO to do that, and an artificial recipe just to create a directory seems like an overkill).